test/plugins/scenarios/mux/main.go

```diff
@@ -54,11 +58,48 @@ func health(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("success"))
 }
 
+var upgrader = websocket.Upgrader{}
+
+func ws(w http.ResponseWriter, r *http.Request) {
+	c, err := upgrader.Upgrade(w, r, nil)
+	if err != nil {
+		log.Print("upgrade:", err)
+		return
+	}
+	defer c.Close()
+	for {
+		mt, message, err := c.ReadMessage()
+		if err != nil {
+			log.Println("read:", err)
+			break
+		}
+		log.Printf("recv: %s", message)
+		err = c.WriteMessage(mt, message)
+		if err != nil {
+			log.Println("write:", err)
+			break
+		}
+	}
+}
+
+func connectWs() {
+	u := url.URL{Scheme: "ws", Host: "localhost:8080", Path: "/ws"}
+	log.Printf("connecting to %s", u.String())
+	c, _, err := websocket.DefaultDialer.Dial(u.String(), nil)
+	if err != nil {
+		log.Fatal("dial:", err)
+	}
+	defer c.Close()
+	c.WriteMessage(websocket.TextMessage, []byte("hello from mux test"))
```

Comment:
I think when you connect to the ws, there will generate a new exit span? But I can't see it in the excepted file.

Reply:
It does not generate a new exit span.

Reply:
Do you mean to generate an exit span with commpent id 5005 (http client)?

Reply:
Ok, It's another framework we haven't enhanced, right?

Reply:
Yes, the bottom layer of gorilla does not directly use http.Client, which is the point we are injecting now (*Transport.RoundTrip)

Reply:
OK, could you help to create other PR to enhance this framework? I think this test code should belong to that plugin test is more reasonable, we can put it in there temporarily. 
Also, Could you add a comment to explain why do need to add this framework in this test case?

Reply:
Because this framework will convert `http.ResponseWriter` to `http.Hijacker` when receiving a weboscket connection request, if you think it is redundant, I can directly add a line of forced conversion code to the `provider` method.

Reply:
Oh, I mean write this in the code as a comment. If some develop read this code maybe confuse about it.

Reply:
Sorry.

Reply:
Done.